hr_attendance_split/models/hr_attendance.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 Pavlov Media
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import api, fields, models, _
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT, \
+    DEFAULT_SERVER_DATE_FORMAT
+
+import pytz
+from datetime import datetime, timedelta, time
+from odoo.exceptions import ValidationError
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    split_attendance = fields.Boolean(
+        string="Split Attendance",
+        help="If attendance was split due to overnight coverage.")
+    company_id = fields.Many2one(
+        'res.company',
+        string="Company",
+        related="employee_id.company_id", store=True)
+    related_attendance_id = fields.Many2one('hr.attendance',
+                                            string='Related Attendance')
+
+    def _get_attendance_employee_tz(self, date=None):
+        """ Convert date according to timezone of user """
+        if not date:
+            return False
+        tz = False
+        if self.employee_id:
+            tz = self.employee_id.tz
+        time_zone = pytz.timezone(tz or 'UTC')
+        attendance_dt = datetime.strptime(str(date),
+                                          DEFAULT_SERVER_DATETIME_FORMAT)
+        attendance_tz_dt = pytz.UTC.localize(attendance_dt)
+        attendance_tz_dt = attendance_tz_dt.astimezone(time_zone)
+        attendance_tz_date_str = datetime.strftime(
+            attendance_tz_dt, DEFAULT_SERVER_DATE_FORMAT)
+        return attendance_tz_date_str
+
+    @api.model
+    def create(self, vals):
```

Comment:
I am not sure on what to do here. I think we should allow a user to do this if he is manually creating it on purpose

Reply:
Here's my opinion, if the company wants split attendance then it shouldn't allow them to manual do it either. They would get the warning, then the have to manually create multiple attendances just like it automatically does when a user is clocking in/out on different days. I think if we let them enter any in there then it goes against the purpose of this module. 
If it is decided it's best to leave this out, it's easy to remove it.

Reply:
Okay, we can leave it like this for now then. Let's see if someone else has an opinion on this.